compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Pretty.hs

```diff
@@ -421,7 +421,7 @@ instance Pretty Expr where
     EUpdate upd -> pPrintPrec lvl prec upd
     EScenario scen -> pPrintPrec lvl prec scen
     ELocation loc x
-        | lvl >= PrettyLevel 1 -> sep ["@location" <> parens (pretty loc), nest 2 (parens (pPrintPrec lvl 0 x))]
+        | lvl >= PrettyLevel 1 -> prettyAppDoc lvl prec ("@location" <> parens (pretty loc)) [TmArg x]
```

Comment:
Just to be clear, my “simplification” you mean that this simplifies the result not only the code right? This definitely seems to change how things are printed so “simplification” seems somewhat misleading.

Reply:
What I had before was supposed to do what we do now but it didn't.